electron/src/components/update-available/update-available.component.scss

```diff
@@ -1,5 +0,0 @@
-:host {
```

Comment:
I'd rather not show the whole component, since I couldn't found any reason why we should.